plugins/chat/assets/javascripts/discourse/components/chat-footer.gjs

```diff
@@ -0,0 +1,83 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import concatClass from "discourse/helpers/concat-class";
+import i18n from "discourse-common/helpers/i18n";
+import eq from "truth-helpers/helpers/eq";
+
+export default class ChatFooter extends Component {
+  @service router;
+  @service chat;
+  @service chatApi;
+
+  @tracked threadsEnabled = false;
+
+  constructor() {
+    super(...arguments);
+    this.userThreadCount();
+  }
+
+  async userThreadCount() {
+    await this.chatApi.userThreadCount().then((result) => {
+      this.threadsEnabled = result.thread_count > 0;
+    });
+  }
+
+  get directMessagesEnabled() {
+    return this.chat.userCanAccessDirectMessages;
+  }
+
+  get shouldRenderFooter() {
+    return this.directMessagesEnabled || this.threadsEnabled;
+  }
+
+  <template>
+    {{#if this.shouldRenderFooter}}
+      <nav class="c-footer">
+        {{#if this.directMessagesEnabled}}
+          <DButton
+            @route="chat.direct-messages"
+            @class={{concatClass
+              "btn-flat c-footer__item"
```

Comment:
That's very minor, but when using `concatClass`, I prefer to have a string for each class name:

```
{{concatClass "btn-flat" "c-footer__item" ...}}
```